gradle.properties

```diff
@@ -22,7 +22,7 @@ guava_version = 15.0
 httpcomponents_version = 4.3.4
 httpcomponents_client_version = 4.3.6
 httpcomponents_core_version = 4.3.2
-jackalope_version = 2.0.0
+jackalope_version = 3.0.1
```

Comment:
We can also update the README#Library Attribution section to update the version we use.